deploying/deploying.md

```diff
@@ -193,4 +193,4 @@ All done! Small differences in the way you've set up your site may make bits of
 
 - [Tutorial - Heroku Dev Center Deployment](https://devcenter.heroku.com/articles/deploying-nodejs)
 - [Overview of Deployment Options - MDN Express & Node Deployment](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/deployment)
-- [Code School Tutorial(requires sign-in) - Domain Names,Name Servers, and Cloud Based Servers](https://www.codeschool.com/beginners-guide-to-web-development/deploying-your-first-website)
+- [Code School Tutorial(requires sign-in) - Domain Names,Name Servers and Cloud Based Servers](https://www.codeschool.com/beginners-guide-to-web-development/deploying-your-first-website)
```

Comment:
This link is redirecting to `https://www.pluralsight.com/paths/html5` so better to switch it with that or find other one.